composer.json

```diff
@@ -41,13 +41,15 @@
     "roave/security-advisories": "dev-master"
   },
   "require-dev": {
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.3",
+    "php": "^5.6|^7",
+    "brain/monkey": "^2.2",
```

Comment:
Something critical in 2.1 or 2.2 that's needed? (new feature, important bug fix)

Reply:
Nope, nothing critical. Per [BM changelog](https://github.com/Brain-WP/BrainMonkey/releases), the minor release is for Mockery.